apps/teams-test-app/e2e-test-data/nestedAppAuth.json

```diff
@@ -8,6 +8,13 @@
       "type": "callResponse",
       "boxSelector": "#box_checkIsNAAChannelRecommended",
       "expectedTestAppValue": "NAA channel is recommended"
+    },
+    {
+      "title": "nestedAppAuth get parent origin",
+      "type": "callResponse",
+      "boxSelector": "#box_getParentOrigin",
+      "platformsExcluded": ["iOS", "Android"],
```

Comment:
could we please a comment here on why ios/android is excluded? thanks

Reply:
Two reasons:
1. Mobile doesn't need this value because there is no requirement of bridge for mobile.
2. parent origin is nil for mobile because of frameless window.